application/cms/models.py

```diff
@@ -937,4 +939,71 @@ class LowestLevelOfGeography(db.Model):
     description = db.Column(db.String(255), nullable=True)
     position = db.Column(db.Integer, nullable=False)
 
-    pages = relationship("Page", back_populates="lowest_level_of_geography")
+    pages = db.relationship("Page", back_populates="lowest_level_of_geography")
+
+
+class Topic(db.Model):
+    __tablename__ = "topic"
+
+    id = db.Column(db.Integer, primary_key=True)
+    uri = db.Column(db.String(64), nullable=False)
+    title = db.Column(db.String(255), nullable=False)
+    description = db.Column(db.Text, nullable=True)  # a sentence below topic heading on homepage
+    additional_description = db.Column(db.TEXT, nullable=True)  # short paragraph displayed on topic page
+
+    # Find a list of subtopics that belong to this topic using this relationship
+    subtopics = db.relationship("Subtopic", back_populates="topic")
+
+
+subtopic_measure = db.Table(
```

Comment:
Might be nice if this was between subtopic and measure tables.

Reply:
It can't be. Because then it isn't defined in time for when it's used in the relationship.

Reply:
OK I stringified the thing at moved it in-between